walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -13,11 +13,117 @@
 // limitations under the License.
 
 package com.google.sps;
-
+import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.Collection;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Iterator;
+import java.util.List;
 
 public final class FindMeetingQuery {
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
```

Comment:
Suggestion about this method: it's a pretty long method. Think about how you might be able to split it up into helper methods that each have some small and specific purpose. This could make the code look a lot clearer and cleaner! I really like how you use comments at the beginning of each section--maybe you could split up the method into some of these commented sections, if that makes sense.

Reply:
Separating into helper methods would definitely make my code clearer, thank you.